.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: node_js
+
+node_js:
+  - node
+
+cache:
```

Comment:
I think you can shorten this to get more complete node caching: https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#caching-with-npm

Reply:
Nice, done.